packages/augur-sdk/src/state/getter/Liquidity.ts

```diff
@@ -0,0 +1,82 @@
+import { BigNumber } from 'bignumber.js';
+import * as t from 'io-ts';
+import { DB } from '../db/DB';
+import { Getter } from './Router';
+import { Augur } from '../../index';
+import * as _ from 'lodash';
+
+export const Order = t.type({
+    price: t.string,
+    amount: t.string,
+});
+  
+export const OutcomeOrderBook = t.type({
```

Comment:
Is this OutcomeOrderBook just a different syntax?

Reply:
They are the same data shape, but one is an interface, and the other is defined for type-checking when passed into the getter.